### ISSUE TITLE
TFS “Visual Studio Test” task deletes TRX file after publish

### ISSUE THREAD

**Issue by johnqa** [Microsoft/vsts-tasks#7229]
## Troubleshooting
TRX results files are deleted after publishing but I still need them

## Environment
TFS latest version

## Issue Description

TFS “Visual Studio Test” task version 2.* deletes TRX files after publishing, but I still need them for our custom reporting framework

Would it be possible to have the option for keeping the trx files?

**Comment by prawalagarwal** [comment]
@johnqa trx files are available as run attachments with your test run. You can download the files from there. Please let us know if this did not solve your issue.

**Comment by prawalagarwal** [comment]
@johnqa Are you unblocked?

**Comment by Nekobravo** [comment]
![image](https://user-images.githubusercontent.com/23462665/40460646-c257e8f2-5ed5-11e8-96ef-46386945dc04.png)
 Help!!

**Comment by prawalagarwal** [comment]
@Nekobravo this is a separate issue. Please reach out to us at devops_tools@microsoft.com with your run logs with system.debug set true in build variables.

**Comment by prawalagarwal** [comment]
@johnqa closing this. Please reopen if the issue is not resolved.
